credits.txt

```diff
@@ -230,6 +242,7 @@ contributed to Endless Sky:
   mootootwo
   naehc
   Naminson
+  Nams-2
```

Comment:
These are the same person. No need to list people twice.

Reply:
Is this documented somewhere?

Reply:
The credits are all GitHub account names. Naminson is no longer an account name but Nams-2 is, and the PRs opened by Nams-2 are the ones that I recall Naminson opening.

Reply:
Alright, removed.